explorer/interpreter/value.h

```diff
@@ -36,52 +46,58 @@ class AssociatedConstant;
 // details.
 class Value {
  public:
+  // An X-macro to apply a macro to each kind of Value.
+#define FOR_EACH_VALUE_KIND(X)     \
```

Comment:
All macros should have a CARBON_* prefix
https://google.github.io/styleguide/cppguide.html#Preprocessor_Macros

Reply:
Done.